README.md

```diff
@@ -30,12 +30,12 @@ Alternatively, you can find the package available on
 
 Or if you use [**use-package**](https://www.emacswiki.org/emacs/UsePackage):
 
-```emacs-list
+```emacs-lisp
 (use-package editorconfig
   :ensure t
   :init
-  (add-hook 'prog-mode-hook (editorconfig-mode 1))
-  (add-hook 'text-mode-hook (editorconfig-mode 1)))
+  (add-hook 'prog-mode-hook '(editorconfig-mode 1))
```

Comment:
I just have the following line in my init.el, as it can work as a global mode:

     (editorconfig-mode 1)